install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -882,6 +882,20 @@ data:
 {{- if .Values.hubble.redact.http }}
   # Enables redaction of the http URL query part in flows
   hubble-redact-http-urlquery: {{ .Values.hubble.redact.http.urlQuery | quote }}
+{{- if .Values.hubble.redact.http.headers }}
+  #  Enables redaction of http headers by specifying which headers will not be redacted
+{{- if .Values.hubble.redact.http.headers.allow }}
```

I would swap the comment to be inside the `if` block, so that it only is added to the configmap when the value is actually present :)
```suggestion
{{- if .Values.hubble.redact.http.headers.allow }}
  #  Enables redaction of http headers by specifying which headers will not be redacted
```

---

minor nit here, the description is quite hard to parse. I think the following would be simpler?

`Redact all http headers that do not match this list`